docs/framework/interop/marshaling-classes-structures-and-unions.md

```diff
@@ -284,6 +284,6 @@ typedef struct _MYSTRSTRUCT2
   
 ## See also
 - [Marshaling Data with Platform Invoke](marshaling-data-with-platform-invoke.md)
-- [Platform Invoke Data Types](https://msdn.microsoft.com/library/16014d9f-d6bd-481e-83f0-df11377c550f(v=vs.100))
+- [Platform Invoke Data Types](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/ac7ay120(v=vs.100))
 - [Marshaling Strings](marshaling-strings.md)
-- [Marshaling Arrays of Types](https://msdn.microsoft.com/library/049b1c1b-228f-4445-88ec-91bc7fd4b1e8(v=vs.100))
+- [Marshaling Arrays of Types](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/dd93y453(v=vs.100))
```

Comment:
```suggestion
- [Marshaling Different Types of Arrays](marshaling-different-types-of-arrays.md)
```

Do the same change for the other topics that link to 049b1c1b-228f-4445-88ec-91bc7fd4b1e8